### PR TITLE
Feature/send email notifications

### PR DESCRIPTION
**Pull request by dejandeki123**
sending email notification for registration and renting car

### CODE REVIEW

File: src/main/java/levi9/UserService/controller/UserController.java [hunk 1]
```diff
@@ -31,6 +40,11 @@ public ResponseEntity<UserDto> getUserByUsername(@PathVariable String username){
         return new ResponseEntity<>(userService.loadUserByUsername(username), HttpStatus.OK);
     }
 
+    @GetMapping("/getEmailById/{id}")
```
Comment (Galic-Miroslav) [inline src/main/java/levi9/UserService/controller/UserController.java:42]: makes more sense to be only /email/{id}

**Review by Galic-Miroslav — APPROVED** [review]

File: src/main/java/levi9/UserService/controller/UserController.java [hunk 2]
```diff
@@ -20,9 +21,17 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
+    @Autowired
+    private NotificationServiceApi notificationServiceApi;
+
+    @GetMapping("/notify")
+    public String example2() {return notificationServiceApi.example1();}
```
Comment (Galic-Miroslav) [inline src/main/java/levi9/UserService/controller/UserController.java]: looks like a leftover or bad naming :)

Comment (dejandeki123) [reply src/main/java/levi9/UserService/controller/UserController.java]: solved

File: src/main/java/levi9/UserService/controller/UserController.java (hunk 1 above)
Comment (dejandeki123) [reply src/main/java/levi9/UserService/controller/UserController.java:42]: solved

**Pull request merged**